# Path to your catkin_ws
catkin_path: /home/grvc/catkin_ws

# All the following paths must be relative to catkin_path!

# Path to the digit classifier
model_file: src/text_from_images/model/digit_classification.h5
# Four boxes coordinates to crop the digits: [x0,x1,y0,y1]
boxes:
  [[4, 27, 20, 80],
  [27, 50, 20, 80],
  [50, 73, 20, 80],
  [73, 96, 20, 80]]
# Parameters for each camera:
#  -- int_points: Points in the original image corresponding to corners of the square
#  -- pts_dst: Points in the destiny image after performing homography
#  -- topic digits: Topic in which the image of the digits is published
#  -- output_file_luxes: File in which to write the timestamps along with the read digits
blue:
  int_points: [[0, 0], [0, 0], [0, 0], [0, 0]] # GRADED: Add the points
  pts_dst: [[0, 0], [0, 0], [0, 0], [0, 0]] # GRADED: Add the points
  topic_digits: /usb_cam/image_raw
  output_file_luxes: src/text_from_images/output/Recognized_blue.txt 
elp:
  int_points: [[0, 0], [0, 0], [0, 0], [0, 0]] # GRADED: Add the points
  pts_dst: [[0, 0], [0, 0], [0, 0], [0, 0]] # GRADED: Add the points
  topic_digits: /camera/color/image_raw
  output_file_luxes: src/text_from_images/output/Recognized_elp.txt 
zed:
  int_points: [[322, 150], [324, 61], [409, 71], [409, 157]]
  pts_dst: [[0, 100], [0, 0], [100, 0], [100, 100]]
  topic_digits: /usb_cam/image_raw
  output_file_luxes: src/text_from_images/output/Recognized_zed.txt 
rs:
  int_points: [[632, 209], [634, 75], [762, 88], [762, 220]]
  pts_dst: [[0, 100], [0, 0], [100, 0], [100, 100]]
  topic_digits: /usb_cam/image_raw
  output_file_luxes: src/text_from_images/output/Recognized_rs.txt 
ecap:
  int_points: [[630, 62], [630, 198], [760, 76], [760, 209]]
  pts_dst: [[0, 0], [0, 100], [100, 0], [100, 100]]
  topic_digits: /usb_cam/image_raw
  output_file_luxes: src/text_from_images/output/Recognized_ecap.txt 

